ReactAndroid/src/main/java/com/facebook/react/modules/appearance/AppearanceModule.java

```diff
@@ -66,6 +67,17 @@ private String colorSchemeForCurrentConfiguration(Context context) {
     return "light";
   }
 
+  @Override
+  public void setColorScheme(String style) {
+    if ("dark".equals(style)) {
+      AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
+    } else if ("light".equals(style)) {
+      AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
+    } else if ("unspecified".equals(style)) {
```

Comment:
ColorSchemeName e.g. as specified in TS typings doesn't allow "unspecified".

Maybe we could accept null to reset?

Reply:
The **`Appearance`** module accepts nullable `ColorSchemeName`.

```ts
type ColorSchemeName = 'light' | 'dark' | null | undefined;
```

That value is then converted into `'light' | 'dark' | 'unspecified'` and passed into turbomodule spec **`NativeAppearance`** where it gets converted into its native value counterpart.

I do this so **iOS** `RCT_CONVERT` is more natural.

**Android** suffers because it's all over the place with naming the feature anyway. They now have something called "MODE_NIGHT_UNSPECIFIED", but works for localNightMode only. Sigh.